src/pages/tx-details/tx-details.ts

```diff
@@ -0,0 +1,53 @@
+import { Component } from "@angular/core";
+import { NavParams } from 'ionic-angular';
+import { TxFormatProvider } from '../../providers/tx-format/tx-format';
+import { WalletProvider } from '../../providers/wallet/wallet';
+import { ExternalLinkProvider } from '../../providers/external-link/external-link';
+
+@Component({
+  selector: 'page-tx-details',
+  templateUrl: 'tx-details.html'
+})
+export class TxDetailsPage {
+  public title: string;
+  public tx: any;
+  public destinationAddress: string;
+
+  private wallet: any;
+  
+  constructor(
+    private navParams: NavParams,
+    private txFormatProvider: TxFormatProvider,
+    private walletProvider: WalletProvider,
+    private externalLinkProvider: ExternalLinkProvider,
+  ) {
+    this.wallet = this.navParams.data.wallet;
```

Comment:
Same as above. Only receive `walletId`, then this.profileProvider.getWallet(id)

Reply:
+1

Reply:
#82e71c1